### PR TITLE
Set up a "Hello world" project with two apps (Front end with react)

### PR DESCRIPTION
**Pull request by quavo19**
### In this milestone, I;

- Created an App component with react-router.

- Created a Greeting component that will display a greeting.

- Created a store, an action, and a reducer that will connect to my API endpoint to get the random greeting.

- Displayed the random greeting in my Greeting component.

### CODE REVIEW

File: src/Greeting.js [hunk 1]
```diff
@@ -0,0 +1,24 @@
+import React, { useEffect, useState } from 'react';
+
+const Greeting = () => {
+  const [greeting, setGreeting] = useState('');
+
+  const fetchGreeting = async () => {
+    const response = await fetch('http://localhost:3000/greetings/random');
+    const data = await response.json();
+    setGreeting(data.greeting);
+  };
```
Comment (GzimAsani) [inline src/Greeting.js]: - [ ] The fetching of the api should be done on the Redux

File: src/store/store.js [hunk 2]
```diff
@@ -0,0 +1,6 @@
+import { createStore } from 'redux';
```
Comment (GzimAsani) [inline src/store/store.js]: - Create store is deprecated I suggest you use Toolkit

**Review by GzimAsani — CHANGES_REQUESTED** [review]
### Hello, @quavo19                    :wave:   
 Good job so far,  done There are some issues that you still need to work on to go to the next project but you are almost there!
 
 ## Highlights
 * Good job adding the App and Greetings button

 ## Required Changes 

- [ ]Check the comments below :arrow_double_down: 

 ## Optional suggestions
 _Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

- N/A



Cheers and Happy coding! clapclapclap
 
 Feel free to leave any questions or comments in the PR thread if something is not 100% clear. Please, remember to tag me in your question so I can receive the notification.
 
 **_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**
 
 _As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by quavo19** [comment]
@GzimAsani sure, your review is well understood, I'm working on it now

**Review by Qoosim — APPROVED** [review]
#  *Status: Approved* :tada: :heavy_check_mark: 

Hi @quavo19, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Highlights
-  No linter errors. :heavy_check_mark: 
-  Good README file. :heavy_check_mark: 
-  Correct Gitflow. :heavy_check_mark: 



## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**